src/main/resources/META-INF/constraints/solvers

```diff
@@ -1,2 +0,0 @@
-gov.nasa.jpf.constraints.solvers.dontknow.DontKnowSolverProvider
-gov.nasa.jpf.constraints.smtlibUtility.solver.SMTLibExportSolverProvider
```

Comment:
Is there a reason why the SMTLibExportSolver hast not survived in the new `src/main/resources/META-INF/services/gov.nasa.jpf.constraints.solvers.ConstraintSolverProvider` file, but the DontKnowSolver has?

Reply:
When I first checked the solvers, I was still on the master tree, which only has the DontKnowSolver. I didn't double check if there were new solvers in the develop branch. My fault.